src/Broker/Broker.c

```diff
@@ -57,7 +63,8 @@ Broker_deliver_request(const enum RemoteInterface interface, const uint8_t* cons
     const Packetizer_PacketType packet_type = Packetizer_PacketType_Telemetry;
 #endif
 
-    const size_t packet_size = Packetizer_packetize(&packetizer_data,
+    packetizer_packetize_function packetizer_packetize = packetizers[selected_packetizer].packetize;
```

Comment:
We need to talk about this, because I expected this lookup to be based on bus_id derived from the interface, not the selected_packetizer coming from initialize....

Reply:
done